fms_core/template_paths.py

```diff
@@ -18,5 +18,5 @@
 CONTAINER_MOVE_TEMPLATE = static("submission_templates/Container_move_v0.4.xlsx")
```

Comment:
Just make sure the sample submission template is not an empty file as mentioned above ...